[2022-02-02 07:29:07,666] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: gcp_ingest_zones.local_to_gcs_task scheduled__2022-02-01T00:00:00+00:00 [queued]>
[2022-02-02 07:29:07,706] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: gcp_ingest_zones.local_to_gcs_task scheduled__2022-02-01T00:00:00+00:00 [queued]>
[2022-02-02 07:29:07,707] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 07:29:07,709] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-02 07:29:07,711] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 07:29:07,734] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-02-01 00:00:00+00:00
[2022-02-02 07:29:07,747] {standard_task_runner.py:52} INFO - Started process 226 to run task
[2022-02-02 07:29:07,755] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'gcp_ingest_zones', 'local_to_gcs_task', 'scheduled__2022-02-01T00:00:00+00:00', '--job-id', '589', '--raw', '--subdir', 'DAGS_FOLDER/dag_***_zones.py', '--cfg-path', '/tmp/tmp95mix7a3', '--error-file', '/tmp/tmpu4o8dced']
[2022-02-02 07:29:07,757] {standard_task_runner.py:77} INFO - Job 589: Subtask local_to_gcs_task
[2022-02-02 07:29:07,915] {logging_mixin.py:109} INFO - Running <TaskInstance: gcp_ingest_zones.local_to_gcs_task scheduled__2022-02-01T00:00:00+00:00 [running]> on host 34481340dc44
[2022-02-02 07:29:08,017] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gcp_ingest_zones
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-01T00:00:00+00:00
[2022-02-02 07:29:08,763] {python.py:175} INFO - Done. Returned value was: None
[2022-02-02 07:29:08,777] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=gcp_ingest_zones, task_id=local_to_gcs_task, execution_date=20220201T000000, start_date=20220202T072907, end_date=20220202T072908
[2022-02-02 07:29:08,818] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 07:29:08,873] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
